pkgs/by-name/pl/plasma-panel-colorizer/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  kdePackages,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "plasma-panel-colorizer";
+  version = "0.5.2";
+
+  src = fetchFromGitHub {
+    owner = "luisbocanegra";
+    repo = "plasma-panel-colorizer";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-+JweNB+zjbXh6Htyvu2vgogAr5Fl5wDPCpm6GV18NJ0=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    kdePackages.extra-cmake-modules
+    kdePackages.plasma-desktop
```

Comment:
plasma-desktop should probably go to buildInputs

Reply:
plasma-desktop is only needed for building, assuming that the user is already using kde plasma